Example/package.json

```diff
@@ -67,12 +67,11 @@
         "qrcode": "1.4.4",
         "react": "17.0.2",
         "react-dom": "17.0.2",
-        "react-native": "0.65.1",
-        "react-native-animateable-text": "0.7.0",
+        "react-native": "0.67.3",
```

Comment:
What about `metro-react-native-babel-preset`?

Reply:
Thanks for highlighting, I checked and it's already `0.66.2`, almost the same as RN updater suggest: `"metro-react-native-babel-preset": "^0.66.2",`, though I see on npm that it already has [`0.69.0` version](https://www.npmjs.com/package/metro-react-native-babel-preset) (maybe that's why better to use a cap `^` 🤔 )